demo/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: demo
+---
+
+<div style="margin: 0px 1em 2em 1em;">
+On this page you can play with a live demo of the Cryostat Web UI. This runs entirely in your browser and does not access any real Cryostat backend server instances.
+Core JFR functionality is not available in this stubbed out demo, but you can use this to get a sense of what features Cryostat offers and what it looks like.
+</div>
+
+<div style="position: relative;">
+  <div
```

Comment:
How about:

```diff
diff --git a/_sass/core/demo.scss b/_sass/core/demo.scss
new file mode 100644
index 0000000..51127d0
--- /dev/null
+++ b/_sass/core/demo.scss
@@ -0,0 +1,9 @@
+.linear-dot-spinner{
+    width: 2em;
+    aspect-ratio: 4;
+    background: radial-gradient(circle closest-side, #73BCF7 90%,#0000) 0/calc(100%/3) 100% space;
+    clip-path: inset(0 100% 0 0);
+    animation: linear-dot-spinner-animate 1s steps(4) infinite;
+}
+
+@keyframes linear-dot-spinner-animate {to {clip-path: inset(0 -34% 0 0)}}
diff --git a/assets/css/main.scss b/assets/css/main.scss
index 6a4fc9b..5aebc2f 100644
--- a/assets/css/main.scss
+++ b/assets/css/main.scss
@@ -7,6 +7,7 @@
 
 @import "core/cryostat";
 @import "core/blog";
+@import "core/demo";
 
 @import "includes/redhat-footer";
 @import "includes/back-to-top.scss";
diff --git a/demo/index.md b/demo/index.md
index 5b5ce4e..cd95800 100644
--- a/demo/index.md
+++ b/demo/index.md
@@ -9,9 +9,8 @@ Core JFR functionality is not available in this stubbed out demo, but you can us
 
 <div style="position: relative;">
   <div
-    id="loading"
     style="position: absolute; left: 0; top: 0; width: 100%; height: 100%; align-items: center; display: flex; justify-content: center">
-      Demo loading...
+      <div id="loading" aria-label="Loading demo" class="linear-dot-spinner"></div>
   </div>
   <iframe
     id="demo-frame"
```

This will show a linear spinning loading state similar to what we have for web-client.

![Screenshot from 2023-05-09 13-12-12](https://github.com/cryostatio/cryostatio.github.io/assets/68053619/6f97e82b-e6fd-486c-9f4c-59fd61006044)